locale/cy/LC_MESSAGES/django.po

```diff
@@ -520,18 +520,19 @@ msgid "<address>%(polling_station_address)s</address>"
 msgstr ""
 
 #: wcivf/apps/elections/templates/elections/includes/_polling_place.html:57
-msgid "It will be open from <strong>7am to 10pm</strong>."
-msgstr "Bydd ar agor o <strong>7am tan 10pm</strong>."
+#, python-format
+msgid "It will be open from <strong>%(open_time)s to %(close_time)s</strong>"
+msgstr "Bydd ar agor o <strong>%(open_time)s tan %(close_time)s</strong>"
```

Comment:
I know we've agreed we're going to ditch translations on WDIV and WCIVF
https://app.asana.com/0/1205090401342417/1208714863410843/f
but it was relatively easy to just do this right
so I did.